src/http/StatusCode.h

```diff
@@ -59,32 +60,33 @@ typedef enum {
     scUnsupportedMediaType = 415,
     scRequestedRangeNotSatisfied = 416,
     scExpectationFailed = 417,
-    scMisdirectedRequest = 421,     /**< RFC7540 section 9.1.2 */
-    scUnprocessableEntity = 422,    /**< RFC2518 section 10.3 / RFC4918 */
-    scLocked = 423,                 /**< RFC2518 section 10.4 / RFC4918 */
-    scFailedDependency = 424,       /**< RFC2518 section 10.5 / RFC4918 */
+    scMisdirectedRequest = 421,
+    scUnprocessableEntity = 422,
+    scLocked = 423,
+    scFailedDependency = 424,
+    scTooEarly = 425,
     scUpgradeRequired = 426,
-    scPreconditionRequired = 428,   /**< RFC6585 */
-    scTooManyRequests = 429,        /**< RFC6585 */
-    scRequestHeaderFieldsTooLarge = 431, /**< RFC6585 */
-    scUnavailableForLegalReasons = 451, /**< RFC7725 */
+    scPreconditionRequired = 428,
+    scTooManyRequests = 429,
+    scRequestHeaderFieldsTooLarge = 431,
+    scUnavailableForLegalReasons = 451,
     scInternalServerError = 500,
     scNotImplemented = 501,
     scBadGateway = 502,
     scServiceUnavailable = 503,
     scGatewayTimeout = 504,
     scHttpVersionNotSupported = 505,
-    scVariantAlsoNegotiates = 506,  /**< RFC2295 */
-    scInsufficientStorage = 507,    /**< RFC2518 section 10.6 / RFC4918 */
-    scLoopDetected = 508,           /**< RFC5842 */
-    scNotExtended = 510,            /**< RFC2774 */
-    scNetworkAuthenticationRequired = 511, /**< RFC6585 */
+    scVariantAlsoNegotiates = 506,
+    scInsufficientStorage = 507,
+    scLoopDetected = 508,
+    scNotExtended = 510,
+    scNetworkAuthenticationRequired = 511,
 
     // The 6xx codes below are for internal use only: Bad requests result
     // in scBadRequest; bad responses in scGatewayTimeout.
 
-    scInvalidHeader = 600,          /**< Squid header parsing error */
-    scHeaderTooLarge = 601         /* Header too large to process */
+    scInvalidHeader = 600, /**< Squid header parsing error */
+    scHeaderTooLarge = 601 /* Header too large to process */
```

Comment:
Since this PR updates code comments, I suggest adjusting Doxygen formatting for these two:

```suggestion
    scInvalidHeader = 600, ///< Squid header parsing error
    scHeaderTooLarge = 601 ///< Header too large to process
```

I do not insist on this change.